server/src/action/build/stderr.js

```diff
@@ -1,6 +1,66 @@
 const AbstractAction = require('../abstract')
 const { param, body } = require('express-validator/check')
 
+/**
+ * @api {put} /api/build/:id/stderr [BUILDER ONLY] Write stderr
+ * @apiVersion 1.0.0
+ * @apiGroup Build
+ * @apiName BuildStderr
+ *
+ * @apiDescription Append error logs from stderr
+ *
+ * @apiParam  (Params) {Integer}  id    Id of the build
+ * @apiParam  (Body)   {String}   data  Error logs to append
+ *
+ * @apiSuccess  {Integer}                               id            Id of the build
+ * @apiSuccess  {Integer[]}                             manifest_id   Ids of manifests to build
+ * @apiSuccess  {Integer}                               [exit_status] Exit status code
+ * @apiSuccess  {String}                                stdout        Logs from stdout
```

Comment:
Each time you append data to `stdout`/`stderr`, you get the whole `stdout`/`stderr` back? :o

Reply:
Yes the library `Bookshelf` fetch the full object on `build = await build.refresh()` calls. Then the object is sent via the websocket to the client.
In case it is to slow we might consider not calling `refresh` to not send another query and just send the log to the front which will concat the log too.